tests/compilation_tests/database_test.py

```diff
@@ -177,7 +184,8 @@ def test_update_data(self):
                     self.assertEqual(ans, database.get_data(name), name)
```

Comment:
Should this one also be ans instead of name?

Reply:
name o7